### PR TITLE
refactor: reformat lists_test.py by pylint

### PR DESCRIPTION
**Pull request by noritakaIzumi**
## fixed:

- trim whitespaces in the end of line
- trim blanklines between pytest tasks (PEP 8)

### CODE REVIEW

**Review by BethanyG — APPROVED** [review]
@noritakaIzumi -- Thank you for this!  🌟

**Comment by BethanyG** [comment]
Humm, the CI is failing for this.  Investigating.

**Comment by noritakaIzumi** [comment]
Is it ok if I tried to add a row to `requirements.txt` ?

The runner fails and prints the following in "canonical_sync":
```text
ModuleNotFoundError: No module named 'toml'
```

I saw the module `toml` not in `requirements.txt`🤔

**Comment by noritakaIzumi** [comment]
Pytest have released a new version yesterday.
And the module `toml` (missing module) is replaced with `tomli`

https://github.com/pytest-dev/pytest/releases/tag/7.0.0#:~:text=Switch%20TOML%20parser%20from%20toml%20to%20tomli

I think we have to specific version of pytest in `canonical_sync`.

In other words

`.github/workflows/ci-workflow.yml` line 69:

```yaml
      run: pip install pytest
```

needs to be this

```yaml
      run: pip install pytest~=6.2.2
```

**Comment by BethanyG** [comment]
@noritakaIzumi   Thank you for the offer of help.  😄    

This is not _exactly_  about a missing requirement -- at least not one in `requirements.txt`.  When the CI runs, it builds our [test runner](https://github.com/exercism/python-test-runner) by pulling _it's_ `requirements.txt` and making a docker container.  Then, from inside the container, it runs a script located in `python/bin` named `test_exercises.py`.  

`test_exercises.py` imports `data.py` from the same directory.  It's the `data.py` file that is failing on the `toml` import error.  But it **_shouldn't_** be having that issue - so I think the error may be tagging off of something incorrect.  That import has been there a looong time.  It only started failing today, PST.  Additionally, the same script running on my desktop is succeeding.

What **did** change today was the release of PyTest 7.0 (_we were using 6.2.5_).  And since our `requirements.txt` only specifies a minimum version and not a max, the CI container got built with PyTest 7.0.  Since that is what has most recently changed,  I am attempting to confirm by testing on my machine.  And I would love help -- if you are familiar/comfortable with docker and running pytest, et. al. 🌟 

Basically, I am trying to figure out what changed to make the CI throw a fit over a toml import that has not changed.  My working theory is that PyTest 7.0 has caused some sort of issue, that is manifesting with that error.  But right now, I can't figure it out.  😢   I'm having issues getting the script to run manually from docker.  

If it **is** PyTest 7.0, we can fix it by changing the `requirements.txt` for the _runner_, so that it forces PyTest 6.2.5....if that makes sense?

**Comment by BethanyG** [comment]
@noritakaIzumi  -- didn't see your second comment -- YES!  I think you are correct there -- we need to pin the version until we can figure out why PyTest 7.0 is making a mess of things.

**Comment by noritakaIzumi** [comment]
@BethanyG I think so too.

**Comment by BethanyG** [comment]
_**and just like that!**_  😄   🎉 

@noritakaIzumi  -- care to PR a change to the runner `requiremnts.txt`, so that we make sure it doesn't pull PyTest 7.0 by accident?  😉    The repo is here:  https://github.com/exercism/python-test-runner.

**Comment by noritakaIzumi** [comment]
@BethanyG I got it 🥰
